include/NovelRT/Experimental/Input/Input.h

```diff
@@ -0,0 +1,32 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_EXPERIMENTAL_INPUT_H
+#define NOVELRT_EXPERIMENTAL_INPUT_H
+
+//Input Dependencies
+#include <map>
+#include <string>
+#include "../../LoggingService.h"
+#include "../../PluginManagement/PluginManagement.h"
+#include "../../Timing/Timestamp.h"
+
+/**
+ * @Brief The experimental input plugin API.
+ */
+namespace NovelRT::Experimental::Input
+{
+    class NovelKey;
+    enum class KeyState;
+    struct InputAction;
+    class IInputService;
```

Comment:
Could we reorganise these to be sorted by type? i.e. enum, class, struct? (I forget which order I used in the ECS graphics header).

Reply:
Tried to reorganize what I found with multiple types